google/cloud/sql/connector/connector.py

```diff
@@ -65,5 +67,6 @@ def connect(instance_connection_string, driver: str, **kwargs):
     # Return a DBAPI connection
 
     loop = _get_loop()
-    icm = InstanceConnectionManager(instance_connection_string, loop)
+    user_agent = f"cloud-sql-python-connector-{driver}/{__version__}"
```

Comment:
Is the end goal to have different artifacts for `cloud-sql-python-connector` in each driver? Or will it be just one artifact to download and install? 

If it's one, we should consider the driver as a metadata tag instead?

Reply:
We could append it as version metadata instead.

Also, I'm actually not sure if we should release one or multiple artifacts. The issue with releasing one artifact is that you end up installing unnecessary drivers when you only need one. Another solution would be to have users install the driver separately but I can see there being a lot of issues with version incompatibilities.